packages/admin/src/components/Manager/ScraperManager/style.ts

```diff
@@ -0,0 +1,124 @@
+import { css } from "@emotion/css";
+import { colors } from "@shared/styles/color";
+import { hashClassNames } from "src/lib/hash";
+
+export default () => {
+  const {
+    managerHeader,
+    managerTitle,
+    buttonBox,
+    button,
+    tooltip,
+    play,
+    pause,
+    stop,
+    replay,
+    disabled,
+    managerBox,
+  } = hashClassNames([
+    "managerHeader",
+    "managerTitle",
+    "buttonBox",
+    "button",
+    "tooltip",
+    "play",
+    "pause",
+    "stop",
+    "disabled",
+    "replay",
+    "managerBox",
+  ]);
+
+  const manager = css`
+    max-width: 84rem;
+    margin: 0 3rem 4.5rem 3rem;
+
+    .${managerHeader} {
+      display: flex;
+      height: 2.3rem;
+      margin: 0 0 5rem 0;
+      .${managerTitle} {
+        height: fit-content;
+        font-size: 2.3rem;
+        font-weight: 400;
+      }
+      .${buttonBox} {
+        margin-left: auto;
+        .${button} {
+          height: 2.3rem;
+          margin-left: 0.5rem;
+          padding: 0;
+          border: none;
+          outline: none;
+          background-color: transparent;
+          font-size: 2.3rem;
+          cursor: pointer;
+          &:first-child {
+            margin-left: 0;
+          }
+          .${tooltip} {
+            visibility: hidden;
+            z-index: 1;
+            position: absolute;
```

Comment:
```suggestion
            position: absolute;
            bottom: 3rem;
            left: -2rem;
```

왼쪽 버튼의 툴팁이 오른쪽 버튼을 가려서 버튼을 누르기 위해 마우스를 위로 올렸다가 다시 누르는 게 살짝 불편한 것 같습니다. 혹시 툴팁이 버튼 오른쪽이 아니라 위쪽에 뜨는 건 어떻게 생각하시나요?
(rem 단위는 눈대중으로 맞췄습니다. 무시하셔도 돼요! 😂)
<img width="172" alt="스크린샷 2022-01-10 오후 4 44 39" src="https://user-images.githubusercontent.com/71015915/148732785-f2f5921f-15eb-4ad9-be1e-5eeb58e2de68.png">